expression/builtin_arithmetic_vec.go

```diff
@@ -814,11 +813,136 @@ func (b *builtinArithmeticDivideRealSig) vecEvalReal(input *chunk.Chunk, result
 }
 
 func (b *builtinArithmeticIntDivideIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinArithmeticIntDivideIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	lh, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(lh)
+
+	if err := b.args[0].VecEvalInt(b.ctx, input, lh); err != nil {
+		return err
+	}
+
+	// reuse result as rh to avoid buf allocate
+	if err := b.args[1].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+
+	result.MergeNulls(lh)
+
+	rh := result
```

Comment:
it's better to `s/lh/lhs/` and `s/rh/rhs/`, for `lhs` is short for `left hand side` and `rhs` is short for `right hand side`

Reply:
Thanks for your review. Should I rename `lh` to `lhs` and `rh` to `rhs`?

Reply:
yes